content/hub/contributing/index.md

```diff
@@ -39,7 +41,9 @@ Once you're done, you'll want to add a "commit message" which describes your pro
 
 Then click "Propose new file" at the bottom. Then click the big, green "Create pull request" button. Then you'll see a couple more text boxes where you can describe your changes. This is the main post (called a "pull request") which the maintainers will read to understand what you're trying to do. You can use the commit message you already wrote if it's good enough for that. Once you're finished, click "Create pull request" at the bottom and the maintainers will look at it and decide if they want to add your changes. They might also reply with questions or requests for tweaks before they accept it.
 
-## Option 2: Clone the repository
+## Option 2
+
+## Developers: Clone the repository
```

Comment:
the empty section here is a bit odd (Especially for screenreaders.) maybe `Option 2: Developer: Clone the Repository` or `[Option 2] Deveopers: Clone the repo` or so?